modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java

```diff
@@ -2535,4 +2536,19 @@ private void waitUntilStatusToBlock(String apiId, String action) throws APIManag
         }
 
     }
+
+    /**
+     * Get the Certificate usage
```

Comment:
Shall we end this sentence with a dot (period)? Let's try to follow the javadoc standards as much as possible.

Reply:
Fixed https://github.com/wso2/product-apim/pull/13040